packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -0,0 +1,87 @@
+import * as fs from 'node:fs'
+
+import {Command} from '@oclif/core'
```

Comment:
We usually add spaces around the curly brackets, should we modify ESLint to accept it as well?
```suggestion
import { Command } from '@oclif/core'
```

Reply:
I'm more and more thinking we should just remove the ESlint of OCLIF xD
But not sure about the side effect this could have... 🤔

Reply:
I'm not sure, but it's just the linter, then I think it should be fine 😄

Reply:
@hoangmirs it's the same issue @tyrro raised [here](https://github.com/nimblehq/react-templates/pull/101#discussion_r913632393) so how about fixing this on [the shared ESLint](https://github.com/nimblehq/eslint-config-nimble)?

Reply:
I overridden some basic rules for now in [fb959b5](https://github.com/nimblehq/react-templates/pull/95/commits/fb959b5bf65c08d34212162339ca1393c93ed045)

Reply:
that's great, Thanks @carryall !